tests/integration/cloudformation/test_cloudformation_engine.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+#
+TMPL = """
+Resources:
+  blaBE223B94:
+    Type: AWS::SNS::Topic
+  queue276F7297:
+    Type: AWS::SQS::Queue
+    Properties:
+      DelaySeconds: "2"
+      FifoQueue: "true"
```

Comment:
that's basically the core problem of the linked issue

If you have any other requests that are notorious for these kinda conversions we can add a test for these here as well :+1: